flutter_local_notifications/lib/src/platform_specifics/ios/notification_details.dart

```diff
@@ -71,4 +72,38 @@ class IOSNotificationDetails {
   ///
   /// This property is only applicable to iOS 10 or newer.
   final String? threadIdentifier;
+
+  /// The interruption level that indicates the priority and
+  /// delivery timing of a notification.
+  ///
+  /// This property is only applicable to iOS 10 or newer.
+  final InterruptionLevel? interruptionLevel;
```

Comment:
As the plugin supports null-safety, what if this is were to be made non-nullable? My understanding is that `active` is the default interruption level to begin with. If that's the case, making it non-nullable and having `active` as default would be better

Reply:
Sounds reasonable. I suppose it also mimics the fact that even if one were to write their app entirely in Objective C or Swift, that you can omit the interruption level too.